auth-web/src/views/auth/DashboardView.vue

```diff
@@ -68,9 +68,10 @@ export default class DashboardView extends Vue {
     }
 
     // If user is not an active member of a team at all, redirect to create team page
-    if (!this.myOrgMembership) {
+    /* if (!this.myOrgMembership) {
```

Comment:
I think we will still need this check in case a user without an account navigates directly to the Dashboard.

The problem must be that membership is not getting synced after the new account is created.  If we put a `syncMembership` call immediately after creating the account, that may be all we need.

Reply:
i took this out..may be in another PR